### PR TITLE
unskip blockchaincom & remove seq number checks

### PR DESCRIPTION
**Pull request by ttodua**
1) returned ws tests
2) the checkSequence method was incorrect. i think this exchange WS was not working during last 10 months, because it throws exception immediately during initial incoming messages, because checksequence method should checked the seq-num per channel, but globally
3) lastly, on top of it, I do not know/understand why do we need to check the sequence number , because we dont do that on any other exchange (even though some exchanges do have sequencing numbers). it's just some burden - let's say exchange missis/skips/misbehaves one sequence - why should we break/throw exception? it might be not worth to break users' running scripts for such minor misshap, which might not be critical, and even out of our control (exchange should be blamed if such thing happens).

### CODE REVIEW

**Comment by carlosmiei** [comment]
Can you check if the c# side is working as expected?

**Comment by ttodua** [comment]
@carlosmiei I have had to make some tedious debug to find all the reasons happening in C#, and found few things:
1) any exchange we have now in c#, would not work if they use `countedOrderBook(cOB)`  `, because their base's `.reset()` method access the instance's `_asks & _bids` which are always null for `cOB` , and `.reset()` throws exceptions. also, any other method, which access instance's `asks & bids` properties, are treating them as `Asks` and `Bids` type, while casting `CountedAsks` value to `Asks` type ends up with `null`.
So, at this moment, there is no other way other than copying whole `.reset()` method into `cOB` class. 
2) also, we needed to have default value of `snapshot` to be `{}` instead of null (as in other langs)

**Comment by ttodua** [comment]
Same here - travis throws unrelated issue, however, during manual tests it worked on my local.

File: cs/ccxt/ws/OrderBook.cs [hunk 1]
```diff
@@ -260,6 +261,35 @@ public IOrderBook limit()
     }
 
 
+
+
+    public void reset(object snapshot = null)
+    {
+        lock (_syncRoot)
```
Comment (carlosmiei) [inline cs/ccxt/ws/OrderBook.cs:268]: we can keep it for now, but I think we can find a better way without repeating the reset implementation (with the interfaces)

**Comment by carlosmiei** [comment]
DEMO

```
 npm run cli.cs -- blockchaincom watchOrderBook "BTC/USDT"

> ccxt@4.2.51 cli.cs
> dotnet run --project "./cs/cli/cli.csproj" blockchaincom watchOrderBook BTC/USDT

[
  "BTC/USDT"
]
{
  "nonce": null,
  "datetime": "2024-02-27T11:07:22.737Z",
  "asks": [
    [
      56563.22,
      0.01666666,
      1
    ],
    [
      56610.43,
      0.13569606,
      1
    ],
    [
      56610.44,
      0.03333334,
      1
    ],
    [
      56620.96,
      0.05,
      1
    ],
    [
      57091.36,
      0.00199588,
      1
    ],
    [
      72000.0,
      0.00871242,
      1
    ],
    [
      75000.0,
      0.22310845,
      1
    ],
    [
      100000.0,
      0.04236814,
      1
    ]
  ],
  "bids": [
    [
      55954.14,
      0.01184307,
      1
    ],
    [
      55954.13,
      0.01787,
      1
    ],
    [
      50805.0,
      0.00060342,
      1
    ],
    [
      50400.0,
      0.02,
      1
    ],
    [
      50319.0,
      0.000504,
      1
    ],
    [
```

**Pull request merged**